src/app/accounts/destiny-account.ts

```diff
@@ -113,6 +113,14 @@ function couldBeD1Account(destinyAccount: DestinyProfileUserInfoCard | UserInfoC
   );
 }
 
+function formatBungieName(destinyAccount: DestinyProfileUserInfoCard | UserInfoCard) {
+  return (
+    destinyAccount.bungieGlobalDisplayName +
+    (destinyAccount.bungieGlobalDisplayNameCode &&
```

Comment:
isn't this going to append the string `"undefined"` if it's missing?

Reply:
I suppose it will